README.md

```diff
@@ -1,38 +1,145 @@
-# Filter Employee Modal
+# Zadanie Rekrutacyjne - Michał Skrzypczyński
 
-## Description
+Plusy:
+- Wersja live
+- Readme
+- Fajne użycie styled-components z propsami
+- Proptypes
+- Custom hooks, np. z wielkością okna
 
-A simple website that filters employees by filters selected by a user.
-Made for recruitment purposes.
 
-## Preview
+Minusy:
+- config z pracownikami w App.js ?
+- Zero testów
+- Zakomentowany kod - np w useWindowsSize.js
+- Komponent funkcyjny Home, w którym dużo się dzieje, bardzo dużo hooków, zasadne by tu było użycie klasy i wywalenie większości metod na zewnątrz. Tutaj mocno widać brak Reduxa czym skutkuje.
+- Jak jesteśmy przy Home.js:
+    -   funkcja getFilteredWorkers - długa z komentarzami, idealny kandydat do rozbijania na mniejsze, czytelniejsze funkcje. Dlaczego tak robimy? Po pierwsze, mniejsze funkcje są czytelniejsze i łatwiejsze w testowaniu. W dodatku używając ładnych, opisowych nazw możemy przekazać dużo więcej niż zostawiając komentarze. U nas np jest zasada, że komentarze są zakazane (chyba, że jest ekstremalnie potrzebny). Dlatego właśnie, aby zmuszać naszych devów do przemyślenia funkcji, rozbijania i nazwenictwa. 
```

Comment:
Jeśli chodzi o komentarze wyjaśniające kod to spotkałem się raz z pozytywnym odbiorem jego obecności w kodzie i od tamtej pory uznawałem to za dobrą praktykę. Wasze podejście do tego wydaje mi się jednak znacznie lepsze. Odnośnie tej długiej funkcji - dzięki, zdecydowanie przesadziłem z nią :)

Reply:
Kod powinien sam się bronić logiką i dobrze przemyslanymi nazwami. Nie ma co się bać długich nazw. Polecam książkę Czysty Kod - Boba Martina. Tam dość obszernie rozprawia się z komentarzami :) W praktyce produkcyjnej Code Review jest obowiazkowe (u nas również), więc Twój kod zawsze ktoś będzie czytał, a potem rozwijał. Stąd musi być łatwy do zrozumienia.